scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py

```diff
@@ -0,0 +1,47 @@
+'''
+This script will generate a cleaned CDC precipitation index data file.
```

Comment:
for this too please consider adding a test with sample input/output files. you can use Natalie's EPA test as reference: https://github.com/datacommonsorg/data/tree/master/scripts/epa/airdata